generic/tests/netperf.py

```diff
@@ -264,12 +277,16 @@ def _pin_vm_threads(vm, node):
             tmp = vm_client.wait_for_login(timeout=login_timeout)
             client_ip = vm_client.wait_for_get_address(0, timeout=5)
         elif client != "localhost":
+            client_pub_ip = params.get("client_public_ip")
             tmp = remote.wait_for_login(params.get("shell_client_client"),
-                                        client_ip,
+                                        client_pub_ip,
                                         params.get("shell_port_client"),
                                         params.get("username_client"),
                                         params.get("password_client"),
                                         params.get("shell_prompt_client"))
+            cmd = "ifconfig %s %s up" % (params.get("client_physical_nic"),
+                                         client_ip)
+            ssh_cmd(tmp, cmd)
```

Comment:
@quanwenli Seems this cmd will be executed twice. And it's a linux only cmd, might need to add a limitation.

Reply:
@pezhang it's as expect, as it comment out "# client session 1 for control, session 2 for data communication". actually it only can be linux since it's not vm, not localhost. anyway i add a limitation. please help review. thanks

Reply:
@quanwenli ok, thanks for explain.

Reply:
ok, thanks for explain.